1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+              </template>
+            </test-fixture>
+            <script>
+              suite('<seed-element>', function() {
+                var myEl;
+                setup(function() {
+                  myEl = fixture('seed-element-fixture');
+                });
+                test('defines the "author" property', function() {
+                  assert.equal(myEl.author.name, 'Dimitri Glazkov');
+                });
+              });
+            </script>
+          </body>
+        </html>
+
+1.  Go to the base directory of your element and run your tests.
+
+        wct
+
+    ![output from successful wct unit test run](/1.0/images/wct-output.png)
+
+    `wct` automatically finds all of the browsers on your system and runs
+    your tests against each one.
+
+## Create stub methods
+
+Stubs enable you to replace default implementations with custom methods. This
+is useful for catching side effects. 
+
+    setup(function() {
+      stub('paper-button', {
+        click: function() {
+          console.log('paper-button.click called');
+        }
+      });
+    });
+
+You don't have to use stubs directly on individual elements. You can override
+the implementation for all elements of a given type. 
+
+## Create stub elements
+
+Use `replace()` to create stub elements. 
+
+    setup(function() {
+      replace('paper-button').with('fake-paper-button');
+    });
+
+For example, using the sample `replace()` above and the element below:
+
+    <dom-module id='x-el'>
+      <template>
+        <paper-button id="pb">button</paper-button>
+      </template>
+    </dom-module>
+
+At test runtime, the content template would be stamped out as:
+
+    <dom-module id='x-el'>
+      <template>
+        <fake-paper-button id="pb">button</fake-paper-button>
+      </template>
+    </dom-module>
+
+The attributes and content of the element are preserved, but the tag
+is replaced with the specified stub tag.
+
+Because the method is called within `setup()`, all of the changes are 
+reverted at the end of each test.
+
+## AJAX
+
+<div class="yt-embed">
+  <google-youtube
+    videoid="_9qARcdCAn4"
+    autoplay="0"
+    rel="0"
+    fluid>
+  </google-youtube>
+</div>
+
+`wct` includes [Sinon](http://sinonjs.org/), which enables you to mock XHR
+requests and create fake servers. 
+
+Below is an example of a simple XHR unit test suite for 
+[`<iron-ajax>`](https://elements.polymer-project.org/elements/iron-ajax).
+Check out Sinon's documentation for more in-depth examples. 
+
+{% highlight html %}
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" 
+        content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <!-- import the element -->
+  <link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
+</head>
```

Comment:
Do you need all of this markup or could you shorten this down to just the test fixture and the script block?